.Gruntfile.babel.js

```diff
@@ -111,7 +111,7 @@ module.exports = (grunt) => {
         cmd: './node_modules/.bin/mocha --require Gruntfile.js -R spec --bail "{./{,!(node_modules)/**/}*.test.js,./test/*.js}"',
         options: {env: {LOAD_NO_FILE: 'true', ...process.env}}
       },
-      standard: './node_modules/.bin/standard "**/*.{js,vue}"',
+      standard: './node_modules/.bin/standard "**/*.{js,vue}" --fix',
```

Comment:
I feel like this isn't the first time I'm making this comment about this same addition, but to be clear — `--fix` **must** be run manually. It cannot be run automatically every single time a file changes.

I hope the security and bug-related implications of that are clear and I do not need to spell them out. It is already a security risk that we're using `standard` at all.

Reply:
I perfectly understand and I'm aware of it. I'm sorry, I did stage the file without noticing it, I use it this way only locally. I undid the change.